katcp/resource_client.py

```diff
@@ -29,8 +28,7 @@
 # TODO NM 2017-04-13 Importing IOLoopThreadwrapper here for backwards
 # compatibility, user code should be changed to import it from the more logical
 # katcp.ioloop_manager module.
-from katcp.ioloop_manager import (IOLoopThreadWrapper,
```

Comment:
Deleting this import causes exceptions.
![image](https://user-images.githubusercontent.com/7910856/67209597-f3030f80-f417-11e9-9d2e-2589c12218d7.png)


Not sure why https://github.com/ska-sa/katcorelib/blob/master/katcorelib/mkat_session.py#L31+L35, imports `IOLoopThreadWrapper` from `resource_client` instead of `katcp.ioloop_manager` :confused:

Reply:
Coincidentally I was looking at PEP 8 and it says
> Imported names should always be considered an implementation detail. Other modules must not rely on indirect access to such imported names unless they are an explicitly documented part of the containing module's API

Reply:
Ideally, it would be a good idea to update `katcorelib` such that it imports from `ioloop_manager` but by doing that we might end up breaking other things in the chain, creating ripple effects.